client/my-sites/site-settings/form-general.jsx

```diff
@@ -359,6 +360,31 @@ export class SiteSettingsFormGeneral extends Component {
 		);
 	}
 
+	hideActionbar() {
+		const { translate, fields, updateFields } = this.props;
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="site-settings__wpcom_hide_action_bar">
+					{ translate( 'Action Bar visibility' ) }
+				</FormLabel>
+				<ToggleControl
+					id="site-settings__wpcom_hide_action_bar"
+					__nextHasNoMarginBottom
+					label={ translate( 'Hide the Action Bar on the front end of the site.' ) }
+					checked={ !! fields?.wpcom_hide_action_bar }
+					onChange={ ( newValue ) => {
+						updateFields( { wpcom_hide_action_bar: newValue } );
+					} }
+				/>
+				<FormSettingExplanation>
+					<a href="https://en.support.wordpress.com/action-bar/">
+						{ translate( 'Learn more about the Action Bar.' ) }
```

Comment:
Do we really want the entire text to be a link?

Perhaps we could do it similar to the privacy options below, where "Learn more" is the sole link, and we have a more thorough explanation:

![Screenshot 2025-01-16 at 17 01 17](https://github.com/user-attachments/assets/21c07576-8221-4376-8405-01ba9e7d3233)

Reply:
Sounds good and it was interesting to learn about that and the help center. I'll update.